wh-infra/postgres/main.tf

```diff
@@ -38,23 +40,23 @@ resource "aws_db_instance" "asbwarehouse" {
   allocated_storage = 20
   enabled_cloudwatch_logs_exports = [
     "postgresql",
-    "upgrade"]
+  "upgrade"]
   iam_database_authentication_enabled = false
-  storage_type = "gp2"
-  engine = "postgres"
-  engine_version = "11.6"
-  instance_class = var.postgres_server_spec
-  name = "ga_sb_${var.env}_wh_asbwarehouse_db"
-  identifier = "ga-sb-${var.env}-wh-asbwarehouse-db"
-  username = "postgres"
-  password = var.postgres_admin_password
-  port = 5432
+  storage_type                        = "gp2"
```

Comment:
Hi Dave, could you fix the formatter so it does not do this column-style alignment?

Reply:
Hi Egor, I started trying to figure out how to change the formatting options. It turns out my editor is running the terraform fmt command to apply style changes. The terraform fmt command is not configurable and the layout is part of Terraform's style conventions - https://www.terraform.io/docs/configuration/style.html.